dvc/output.py

```diff
@@ -712,15 +712,10 @@ def verify_metric(self):
         if not self.metric or self.plot:
             return
 
-        if not os.path.exists(self.fs_path):
+        if not os.path.isfile(self.fs_path):
             return
```

Comment:
If we are going to allow a metrics output to be a directory, we need to do the validation checks on the files inside the directory (and walk the directory tree). We cannot skip the checks entirely